### PR TITLE
8.15: update docs, mergify, versions and changelogs

### PR DESCRIPTION
**Pull request by obltmachine**
Merge as soon as the GitHub checks are green.

### CODE REVIEW

File: changelogs/8.15.asciidoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+==== Deprecations
+
+[float]
+==== Intake API Changes
+
+[float]
+==== Added
+
+- Add `elasticsearch.flushed.uncompressed.bytes` metric {pull}13155[13155]
+- APM Server now relies on the Elasticsearch apm-data plugin's index templates, removing the requirement to install the APM integration package {pull}12066[12066]
```
Comment (endorama) [inline changelogs/8.15.asciidoc:31]: Looks a duplicate of [line 15](https://github.com/elastic/apm-server/pull/13552/files#diff-6422fbaf1143429df6cf50366dbf51f7e970a9cad771b57e5a658acae518bfa3R15)? (looks more a new feature to me than a bug fix)

**Review by endorama — CHANGES_REQUESTED** [review]

**Comment by carsonip** [comment]
@kyungeunni this is also missing https://github.com/elastic/apm-server/pull/13554

File: changelogs/8.15.asciidoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
+[[apm-release-notes-8.15]]
+== APM version 8.15
+* <<apm-release-notes-8.15.0>>
+
+[float]
+[[apm-release-notes-8.15.0]]
+=== APM version 8.15.0
+
+https://github.com/elastic/apm-server/compare/v8.14.1\...v8.15.0[View commits]
```
Comment (carsonip) [inline changelogs/8.15.asciidoc:9]:
```suggestion
https://github.com/elastic/apm-server/compare/v8.14.2\...v8.15.0[View commits]
```

See https://github.com/elastic/apm-server/pull/13553

File: changelogs/8.15.asciidoc (hunk 1 above)
Comment (kyungeunni) [reply changelogs/8.15.asciidoc:31]: I think L15 highlights: 
> reverts some unsafe uses of `flattened` field types

from the same commit.

**Comment by kyungeunni** [comment]
> @kyungeunni this is also missing #13554

Added missing entry in https://github.com/elastic/apm-server/pull/13552/commits/f91315e622aa27e541baa7f511dfcc7091320f2c

**Comment by marclop** [comment]
I can't figure out why the docs PR is failing to build

File: changelogs/8.15.asciidoc (hunk 1 above)
Comment (inge4pres) [reply changelogs/8.15.asciidoc:31]: The PR is the same, but I think the same PR was categorized as both bug fix and addition?

**Review by carsonip — APPROVED** [review]

**Comment by inge4pres** [comment]
@endorama do you think we can merge this? I saw you requested changes for the supposedly double occurrence

File: changelogs/8.15.asciidoc (hunk 1 above)
Comment (endorama) [reply changelogs/8.15.asciidoc:31]: thanks for the clarification! Fine by me

**Pull request merged**